packages/cli/src/commands/doctor/types.ts

```diff
@@ -83,6 +89,7 @@ export type RunAutomaticFix = (args: {
 }) => Promise<void> | void;
 
 export type HealthCheckInterface = {
+  [key: string]: any;
```

Comment:
Need to remove this now that is no longer needed.

Reply:
So... can we remove it now?

Reply:
Yes, was a note for me to not forget when I started addressing more feedback